forum/Sources/Subs.php

```diff
@@ -1599,6 +1599,33 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 				'before' => '<span style="color: white;" class="bbc_color">',
 				'after' => '</span>',
 			),
+			array(
+				'tag' => 'video',
+				'type' => 'unparsed_content',
+				'content' => '<iframe width="928" height="522" src="https://www.youtube.com/embed/$1" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>',
+				'validate' => function(&$tag, &$data, $disabled)
+				{
+					$parse = parse_url($data);
+					// Remove 'www.'
+					$url = preg_replace('/^www\./i', '', $parse['host']);
+
+					$query = Array();
+					if ($url == "youtu.be")
+					{
+						$query['v'] = trim($parse['path'], '/');
+					}
+					elseif ($url == 'youtube.com')
+					{
+						parse_str($parse['query'], $query);
+					}
+					else
+					{
+						//Unsupported video, just post URL.
+						$tag['content'] = '<a href=" . $data . "></a>';
```

Comment:
This doesn't work because of the single quotes. The entire value is a string, not concatenated like you might expect.
![screenshot from 2018-11-06 16-35-59](https://user-images.githubusercontent.com/1670549/48100929-76652a00-e1e2-11e8-8670-52f8f9c26cf1.png)